app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -0,0 +1,438 @@
+    <link rel="mask-icon" href="<%= asset_path 'govuk-mask-icon.svg' %>" color="#0b0c0c">
+    <link rel="apple-touch-icon" sizes="180x180" href="<%= asset_path "govuk-apple-touch-icon-180x180.png" %>">
+    <link rel="apple-touch-icon" sizes="167x167" href="<%= asset_path "govuk-apple-touch-icon-167x167.png" %>">
+    <link rel="apple-touch-icon" sizes="152x152" href="<%= asset_path "govuk-apple-touch-icon-152x152.png" %>">
+    <link rel="apple-touch-icon" href="<%= asset_path "govuk-apple-touch-icon.png" %>">
+
+    <meta name="theme-color" content="#0b0c0c" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <% #  Ensure that older IE versions always render with the correct rendering engine %>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <% # The default og:image is added below :head so that scrapers see any custom metatags first, and this is just a fallback %>
+    <meta property="og:image" content="<%= asset_path "govuk-opengraph-image.png" %>">
+
+    <%= yield :head %>
+  </head>
+  <body class="gem-c-layout-for-public govuk-template__body">
+    <script>
+      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+    </script>
+
+    <%= render "govuk_publishing_components/components/skip_link", {
+      href: "#content"
+    } %>
+
+    <%= render "govuk_publishing_components/components/cookie_banner" %>
+
+    <% unless omit_header %>
+      <%= render "govuk_publishing_components/components/layout_header", {
+        environment: "public",
+        search: show_search,
+        # layout-header will always have border-bottom, unless the layout is full width
+        remove_bottom_border: full_width,
+      } %>
+    <% end %>
+    <div class="<%= "govuk-width-container" unless full_width %>"  id="content">
+      <%= yield :before_content %>
+      <main class="govuk-main-wrapper">
+        <%= yield %>
+      </main>
+    </div>
+
+    <%= render "govuk_publishing_components/components/feedback" %>
+
+    <% unless local_assigns[:hide_footer_links] %>
+      <%= render "govuk_publishing_components/components/layout_footer", {
+        with_border: true,
+        navigation: [
+          {
+            title: "Prepare for Brexit",
+            columns: 2,
+            items: [
+              {
+                href: "/business-uk-leaving-eu",
+                text: "Prepare your business or organisation for Brexit",
+                attributes: {
+                  data: {
+                    track_category: "footerClicked",
+                    track_action: "brexitLinks",
+                    track_label: "Prepare your business or organisation for the UK leaving the EU"
+                  }
+                }
+              },
+              {
+                href: "/prepare-eu-exit",
+                text: "Prepare for Brexit if you live in the UK",
```

Comment:
Really weird small/huge thing here - the focus state is clipped on this link. See this comparison with another link.

<img width="667" alt="Screenshot 2020-07-24 at 15 09 41" src="https://user-images.githubusercontent.com/861310/88400164-d5584180-cdbf-11ea-9387-4db9d5531e8f.png">

I think this is something to do with the css columns stuff that's being used here. You can see it's doing some slightly odd wrapping of the element. 

<img width="649" alt="Screenshot 2020-07-24 at 15 11 08" src="https://user-images.githubusercontent.com/861310/88400621-84951880-cdc0-11ea-98f0-4692c7a74722.png">


I thought it would be possible to fix by adding some margin or padding to the bottom of that list item, but the nature of the column wrapping means it just pushes down in the next column. Not sure what to suggest.

Reply:
I've raised this on the Design System slack channel as I believe this issue might be best fixed at the source rather than trying a localised fix on our side. 

It's a really weird one, well done spotting it. I've tried a number of potential solutions but nothing seems to work quite as intended? 😕  

Chrome displays this strange clipping behaviour, but additionally, it looks like other browsers are struggling with this in different ways. 
In Safari for instance, the focus state looks correct, but the links move up and down in a rather jarring way as you tab through the nav items https://gds.slack.com/files/U012K9PKS5N/F017XNZQ3EE/screen_recording_2020-07-29_at_14.52.09.mov

Reply:
This has now been raised as an [issue](https://github.com/alphagov/govuk-frontend/issues/1888) on the Design System team's repo.
@36degrees mentioned that the team are already aware of the issues with CSS `columns` and planning to revisit the header and footer soon. This bug should be resolved as part of that work 🤞